.circleci/config.yml

```diff
@@ -64,45 +64,24 @@ x-run:
   check_changes: &check_changes
     name: Check changes
     command: |-
-      # Skip e2e by default
-      require_e2e=false
+      rc=0
+      scripts/should-e2e-run "${CIRCLE_PROJECT_USERNAME}" "${CIRCLE_PROJECT_REPONAME}" "${CIRCLE_PULL_REQUEST}" || rc=$?
```

Comment:
For some reason the other variable is empty. Looking at the environment where _this_ check is running, that variable is not available. Only CIRCLE_PULL_REQUEST is. So use that and extract the PR number that we need from there.

Reply:
heh - I just got it from their docs page somewhere. fair enough!